pages/linux/lvm.md

```diff
@@ -0,0 +1,36 @@
+# lvm
+
+> Manage physical volumes, volume groups, and logical volumes using the Logical Volume Manager (LVM) interactivce shell.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_logical_volumes/>.
+
+- Start the Logical Volume Manager interactive shell:
+
+`sudo lvm`
+
+- List the Logical Volume Manager commands:
+
+`sudo lvm help`
+
+- Initialize /dev/sda1 to be used as a physical volume:
+
+`sudo lvm pvcreate {{/dev/sda1}}`
```

Comment:
#5036 :thinking:

Reply:
I based the examples on the notation used for the ethtool page I updated. The new notation doesn't seem to have made it into the style guide yet.